sentences/en/media_player_HassSetVolume.yaml

```
@@ -3,6 +3,9 @@ intents:
   HassSetVolume:
     data:
       - sentences:
-          - "set <name> [volume] to <volume>"
+          - "(set|change|increase|decrease) <name> volume to <volume> [percent]"
+          - "Turn <name> volume (up|down) to <volume> [percent]"
```

```suggestion
          - "turn <name> volume (up|down) to <volume>"
```

`<volume>` already is `{volume}[percent]` https://github.com/home-assistant/intents/blob/e5c75313e96a1ffb7008e945d7c784402ff54362/sentences/en/_common.yaml#L341

---

Also, shouldn't `volume` be optional here?

`turn the office speaker up to 11%`